os/board/common/partitions.c

```diff
@@ -63,6 +63,90 @@ FAR struct mtd_dev_s *mtd_initialize(FAR struct mtd_geometry_s *geo)
 #endif
 	return mtd;
 }
+
+void type_specific_initialize(FAR struct mtd_dev_s *mtd_part, int partno, const char *types)
```

Comment:
Please add static for internal functions.

Reply:
Thank you.
I will add static for internal functions.